src/container.c

```diff
@@ -99,3 +126,31 @@ void container_stop(int container_pid) {
   }
   log_debug("container_pid %d killed", container_pid);
 }
+
+char *join_strings(const char *strings[], int start, int end) {
```

Comment:
This might be better for another place, e.g. `utils.c`. While what it does is used by the logic in `container.c`, it has more to do with formatting user input rather than with the container itself